packages/types/src/kilocode/kilocode.ts

```diff
@@ -10,7 +10,7 @@ declare global {
 export const ghostServiceSettingsSchema = z
 	.object({
 		enableAutoTrigger: z.boolean().optional(),
-		autoTriggerDelay: z.number().min(1).max(5000).default(3000).optional(),
+		autoTriggerDelay: z.number().min(1).max(5000).default(200).optional(),
```

Comment:
I'd go with 300 for now, I get the impression that with small numbers (50) the autocompletes sometimes seems to 'choke' so I'd rather have a bit more leeway